speaking-portal-project/src/phonemeFactory/mouthCueProcessor.ts

```diff
@@ -39,7 +41,33 @@ export function generateMouthData(mouthCues: MouthCue[]) {
          etc etc... Will require some tinkering :)
          */
         let dur = (mouthCues[mouthCuesKey].end - mouthCues[mouthCuesKey].start).toFixed(2)
-        let tempStr = `file ../images/${mouthCues[mouthCuesKey].value}.png\noutpoint ${dur}\n`
+        let currentSec = parseInt(mouthCues[mouthCuesKey].start.toFixed(0))
+
+        // TODO: use for testing. After confirming that blinking and other realism is good, delete this
+        console.log("currentSec: %d, dur: %d, lastBlink: %d", currentSec, dur, lastBlink)
+
+        /* BLINKING
+        - 1 blink takes ~0.1s, so only consider frames less than 0.1s long (should we have a minimum?)
+        - Humans typically blink 15-20 times per min, or once every 3-4 seconds
+        - We want it to be random and not happen exactly once every 3 seconds, so let's say...
+            - there's a 40% chance to blink every 4th second
+            - there's a 40% chance to blink every 3rd second
+            - and don't blink in the same second
+        - Image naming scheme:
+            _eyes0 = eyes open
+            _eyes1 = eyes half open
+            _eyes2 = eyes closed
+        */
+        let eyes:string = "_eyes0"
```

Comment:
we should make eyes an eyes type. With the following type definition:

types.ts
export type eyes  = "_eyes0" | "_eyes1" | "eyes2"

mouthCueProcessor.ts
const eye_selection : eyes = "_eyes0"

This might help if we would like to use eye selections throughout the app

Reply:
I'll add this as a separate issue to do next week!

Reply:
Yeah, this would be good. Doing my research people said that ambers' eyes felt to narrow. Maybe switching them could add better quality
\